bugbash.md

```diff
@@ -0,0 +1,2 @@
+# Bug Bash File for activity
+who goes first
```

Comment:
Each word in headings should start with a cap `# Bug Bash File For Activity`

Reply:
Thank you